test/quantization/core/test_quantized_tensor.py

```diff
@@ -328,6 +329,82 @@ def test_qtensor_dtypes(self):
             rqr = qr.dequantize()
             self.assertTrue(np.allclose(r.numpy(), rqr.numpy(), atol=2 / scale))
 
+    @unittest.skipIf(not TEST_CUDA, "No gpu is available.")
+    def test_qtensor_to_device(self):
+        dtype_and_zero_types = [
```

Comment:
nit: you can use itertools.product here I think: https://github.com/pytorch/pytorch/blob/master/test/quantization/fx/test_quantize_fx.py#L559

Reply:
I need to exclude specific combinations which makes a cross product problematic